src/Apps/Order/Routes/Shipping2/Hooks/useSaveSelectedShippingQuote.tsx

```diff
@@ -36,12 +36,21 @@ export const useSaveSelectedShippingQuote = (
     try {
       shippingContext.actions.setIsPerformingOperation(true)
 
+      if (
+        shippingContext.state.fulfillmentDetailsCtx?.values.meta.mode ===
```

Comment:
todo: can this be stored/pulled directly from shippingContext.state and not used in the form ctx?

Reply:
I'm going to leave this for now because it also includes the `pickup` mode, which isn't controlled from outside the component.

Reply:
update: I went ahead and did it. cleaner and also caught an invisible bug.